cherrypy/test/test_static.py

```diff
@@ -376,6 +390,7 @@ def unicode_file():
     )
     @pytest.mark.xfail(py27_on_windows, reason="#1544")
     def test_unicode(self):
+        self.ensure_unicode_filesystem()
```

Comment:
This should probably go to the `unicode_file()` method, so in case of reusing it would always be checked automatically.